docs/API.md

```diff
@@ -108,7 +108,7 @@ Default: `0777`
 
 Octal permission string specifying mode for any folders that need to be created for output folder.
 
-### gulp.task(name[, deps], fn)
+### gulp.task(name [, deps] | [, deps[, fn]] | [, fn])
```

Comment:
The bracket notation means it's optional, nothing to mean that it's an array. Whatever `deps` is, is already explained later.
`gulp.task(name [, deps] | [, deps[, fn]] | [, fn])`
translates to:
`gulp.task(name, deps)`
or
`gulp.task(name, deps, fn)`
or
`gulp.task(name, fn)`